packages/Input/Input.md

```diff
@@ -89,13 +89,17 @@ that the field is optional. It is a more usable and accessible option than the H
 <Input label="Transit number" hint="5 digits" type="number" />
 ```
 
-Use a `helper` message to show additional context that will guide the user while completing the form field.
+Use the `hintPosition` in conjunction with `hint` to display a larger hint text below the label.
 
 ```jsx
-const creditCards = (
-  <Text>
-    We accept the following credit cards: <Text bold>Visa, Mastercard, Discover</Text>.
-  </Text>
-)
-;<Input label="Credit Card Number" helper={creditCards} />
+<Input
+  label="Quick view"
+  hintPosition="below"
+  hint="Enter the account number in the box below to go to the overview page for that account."
+/>
 ```
+
+### Usage criteria
+
+- The `hint` should not be more than 85 characters when positioned below the label
```

Comment:
Consider adding more comprehensive usage criteria, recommending for what character lengths should be using 'inline' vs 'below'.

Reply:
Is there a character cut-off for using `inline` vs `below`? What should be recommended?

Reply:
Updated based on conversation on TDS-1128